common/lib/xmodule/xmodule/js/src/video/09_skip_control.js

```diff
@@ -31,11 +31,9 @@ function() {
             '<button class="control video_control skip skip-control" aria-disabled="false" title="',
                 gettext('Do not show again'),
             '">',
-                '<span class="icon-fallback-img">',
-                    '<span class="icon fa fa-step-forward" aria-hidden="true"></span>',
-                    '<span class="text control-text">',
-                        gettext('Do not show again'),
-                    '</span>',
+                '<span class="icon fa fa-step-forward" aria-hidden="true"></span>',
+                '<span class="text control-text">',
```

Comment:
Since this button has a `title` we can simplify this and remove the `sr` text within it.